packages/main/src/compose/edit.test.ts

```diff
@@ -89,6 +89,64 @@ it("edits a message", () => {
   })
 })
 
+it("has a Content-ID header", async () => {
+  const content = {
+    type: "text",
+    subtype: "plain",
+    content: "What I meant to say was, hello!"
+  }
+  const testMessage = testThread[1].attributes
+  const testPart = testMessage.struct![0] as imap.ImapMessagePart
+  const editedMessage = {
+    envelope_messageId: testMessage.envelope.messageId
+  }
+  const editedPart = {
+    content_id: testPart.id
+  }
+  const { attributes } = composeEdit({
+    account,
+    content,
+    conversation: conversationFrom(testThread),
+    editedMessage,
+    editedPart,
+    resource: {
+      messageId: testMessage.envelope.messageId,
+      contentId: testPart.id
+    }
+  })
+
+  expect(attributes).toMatchObject({
+    struct: [
```

Comment:
I would rather remove some properties from this expected object. I think that just testing for the `id`, `type`, and `subtype` properties would be best. The `id` is the property that is being tested; `type` and `subtype` provide just enough context.